package.json

```diff
@@ -1,100 +1,15 @@
 {
-  "name": "ember-slugify",
-  "description": "Library to slugify your strings within Ember.",
-  "version": "4.0.0",
-  "license": "MIT",
-  "author": {
-    "name": "Dazzling Fugu",
-    "url": "https://github.com/DazzlingFugu"
-  },
-  "homepage": "https://github.com/DazzlingFugu/ember-slugify",
-  "repository": "https://github.com/DazzlingFugu/ember-slugify",
-  "bugs": "https://github.com/DazzlingFugu/ember-slugify/issues",
-  "keywords": [
-    "accent",
-    "accents",
-    "diacritics",
-    "ember",
-    "emberjs",
-    "ember-addon",
-    "simple-pinyin",
-    "slug",
-    "slugify",
-    "string"
-  ],
-  "files": [
-    "/addon",
-    "/app",
-    "/config/environment.js",
-    "/index.js"
-  ],
-  "scripts": {
-    "build": "ember build --environment=production",
-    "lint": "npm-run-all --print-name --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
-    "lint:fix": "npm-run-all --print-name --aggregate-output --continue-on-error --parallel \"lint:*:fix\"",
-    "lint:hbs": "ember-template-lint .",
-    "lint:hbs:fix": "ember-template-lint . --fix",
-    "lint:js": "eslint . --cache",
-    "lint:js:fix": "eslint . --fix",
-    "start": "ember serve",
-    "test": "npm-run-all --print-name \"lint\" \"test:*\"",
-    "test:ember": "ember test",
-    "test:ember-compatibility": "ember try:each"
-  },
-  "dependencies": {
-    "ember-auto-import": "^2.4.3",
-    "ember-cli-babel": "^7.26.11",
-    "ember-cli-htmlbars": "^6.1.1",
-    "emoji.json": "^13.1.0",
-    "simple-pinyin": "^4.0.0",
-    "slugify": "^1.6.5"
-  },
-  "devDependencies": {
-    "@babel/core": "^7.18.2",
-    "@babel/eslint-parser": "^7.18.2",
-    "@ember/optional-features": "^2.0.0",
-    "@ember/test-helpers": "^2.8.1",
-    "@embroider/test-setup": "^2.0.2",
-    "@glimmer/component": "^1.1.2",
-    "@glimmer/tracking": "^1.1.2",
-    "broccoli-asset-rev": "^3.0.0",
-    "ember-cli": "~4.8.0",
-    "ember-cli-dependency-checker": "^3.3.1",
-    "ember-cli-inject-live-reload": "^2.1.0",
-    "ember-cli-sri": "^2.1.1",
-    "ember-cli-terser": "^4.0.2",
-    "ember-disable-prototype-extensions": "^1.1.3",
-    "ember-load-initializers": "^2.1.2",
-    "ember-page-title": "^7.0.0",
-    "ember-qunit": "^6.0.0",
-    "ember-resolver": "^8.0.3",
-    "ember-source": "~4.8.0",
-    "ember-source-channel-url": "^3.0.0",
-    "ember-template-lint": "^5.0.1",
-    "ember-try": "^2.0.0",
-    "eslint": "^8.16.0",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-ember": "^11.1.0",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^4.2.1",
-    "eslint-plugin-qunit": "^7.3.1",
-    "loader.js": "^4.7.0",
-    "npm-run-all": "^4.1.5",
-    "prettier": "^2.7.1",
-    "qunit": "^2.19.2",
-    "qunit-dom": "^2.0.0",
-    "webpack": "^5.74.0"
-  },
+  "private": true,
+  "workspaces": ["ember-slugify", "test-app"],
```

Comment:
When trying to test this branch on another repo I got following error
```
error Can't add "ember-slugify": invalid package version undefined.
```

Do you managed to test it on another repo?

Reply:
I tested it locally with a `yarn link` on an empty app, it worked fine. How did you try it?

Reply:
Cf. discord discussion I was trying by testing the last commit, if as you said it should work when released to NPM LGTM